web-client/src/app/components/balance-summary-card/balance-summary-card.component.html

```diff
@@ -4,23 +4,23 @@
   </ion-card-header>
 
   <ion-card-content>
-    <ion-list lines="none">
-      <ion-grid>
-        <ng-container *ngFor="let balance of balances">
-          <ion-row class="ion-align-items-center">
-            <ion-col size="7" class="ion-text-end">
-              <ion-text color="white" class="text-2xl font-audiowide">
-                {{ balance | assetAmount }}
-              </ion-text>
-            </ion-col>
-            <ion-col size="5" class="ion-text-start">
-              <ion-text color="secondary" class="font-bold">
-                {{ balance | assetSymbol }}
-              </ion-text>
-            </ion-col>
-          </ion-row>
-        </ng-container>
-      </ion-grid>
-    </ion-list>
+    <ion-grid *ngIf="false; else empty">
```

Comment:
Hardcoded `false` for testing: this probably needs replacing?

Reply:
@PiDelport, I've made the suggested changes in the latest commit to this PR. 
@Claire-Bear, the above comment (and requested changes) is reflected in the latest [commit](https://github.com/ntls-io/nautilus-wallet/pull/226/commits/2dfb5a6aaaea9e303a1122a68ef6481a1c88cef1)

Reply:
Looks good!